newIDE/app/src/ProjectManager/index.js

```diff
@@ -641,484 +651,555 @@ export default class ProjectManager extends React.Component<Props, State> {
+                    primaryText={<Trans>Resources</Trans>}
+                    leftIcon={<ArtTrack />}
+                    onClick={this.props.onOpenResources}
+                  />,
+                ]}
               />
-            }
-            initiallyOpen={false}
-            autoGenerateNestedIndicator={true}
-            indentNestedItems
-            renderNestedItems={() => [
-              <ListItem
-                key="properties"
-                primaryText={<Trans>Properties</Trans>}
-                leftIcon={<SettingsApplications />}
-                onClick={this._openProjectProperties}
-              />,
-              <ListItem
-                key="global-variables"
-                primaryText={<Trans>Global variables</Trans>}
-                leftIcon={<VariableTree />}
-                onClick={this._openProjectVariables}
-              />,
-              <ListItem
-                key="icons"
-                primaryText={<Trans>Icons</Trans>}
-                leftIcon={<PhotoLibrary />}
-                onClick={this.props.onOpenPlatformSpecificAssets}
-              />,
-              <ListItem
-                key="resources"
-                primaryText={<Trans>Resources</Trans>}
-                leftIcon={<ArtTrack />}
-                onClick={this.props.onOpenResources}
-              />,
-            ]}
-          />
-          <ProjectStructureItem
-            primaryText={<Trans>Scenes</Trans>}
-            leftIcon={
-              <ListIcon
-                iconSize={24}
-                isGDevelopIcon
-                src="res/ribbon_default/sceneadd32.png"
+              <ProjectStructureItem
+                primaryText={<Trans>Scenes</Trans>}
+                leftIcon={
+                  <ListIcon
+                    iconSize={24}
+                    isGDevelopIcon
+                    src="res/ribbon_default/sceneadd32.png"
+                  />
+                }
+                initiallyOpen={true}
+                open={forceOpen}
+                autoGenerateNestedIndicator={!forceOpen}
+                renderNestedItems={() =>
+                  filterProjectItemsList(enumerateLayouts(project), searchText)
+                    .map((layout: gdLayout, i: number) => {
+                      const name = layout.getName();
+                      return (
+                        <Item
+                          key={i}
+                          primaryText={name}
+                          textEndAdornment={
+                            name === firstLayoutName ? (
+                              <Tooltip
+                                title={i18n._(
+                                  t`This scene will be used as the start scene.`
+                                )}
+                              >
+                                <Flag
+                                  color="disabled"
+                                  fontSize="small"
+                                  style={{
```

Comment:
should this style be integrated in the `<Item>` itself? so that we don't have to copy it again if we want to add a new icon later? (probably not, but it feels cleaner)